Felienne/hedy
# [FEATURE] Adds 'agree to terms' tickmark to sign-up page

**Description**
In this PR we perform an overall UI re-style of the signup page and implement an additional tick mark for agreeing with the privacy agreement. Real-time form validation is implementation using the CSS `input:valid` attribute. A very small re-write is performed in auth.py to better differentiate between functionality within functions and make clear where the mailinglist and teacher account filtering is done.

**Fix for**
This PR fixes #2023 

**How to test**
Navigate to the signup page and notice the new UI and the new tickmark at the bottom of the page. Verify that the tickmark is not required and (when not in testing) an e-mail is sent to the admin when the tickmarked is checked on signup.

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
